<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralTab</class>
 <widget class="QWidget" name="GeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="_useRaster">
     <property name="toolTip">
      <string>Render using &quot;raster&quot;.  This is faster, except for remote X.</string>
     </property>
     <property name="whatsThis">
      <string>Rendering using &quot;raster&quot; is faster unless you are running kst over a remote X connection.</string>
     </property>
     <property name="text">
      <string>Use &quot;raster&quot; engine on (Kst restart needed)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="_transparentDrag">
     <property name="text">
      <string>Use transparency when dragging</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="_antialiasPlots">
     <property name="text">
      <string>&amp;Antialias Plots</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Period between updates (ms):</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>_maxUpdate</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="_maxUpdate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Dynamic files update this fast or slower.</string>
     </property>
     <property name="whatsThis">
      <string>Minimum time between updates when looking at dynamic files.  If there is a lot of processing, it might take longer.</string>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>60000</number>
     </property>
     <property name="singleStep">
      <number>500</number>
     </property>
     <property name="value">
      <number>2000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>44</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>_useRaster</tabstop>
  <tabstop>_transparentDrag</tabstop>
  <tabstop>_maxUpdate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
